spyder/app/tour.py

```diff
@@ -144,68 +144,74 @@ def get_tour(index):
-                           "inline plots). If you want to know more about them, "
-                           "please follow this <a href=\"{qtconsole_link}\">link</a>.<br><br>"
-                           "{button_text}").format(
-                           qtconsole_link=qtconsole_link, button_text=button_text),
+                           "programming workflow, like syntax highlighting, "
+                           "autocompletion, plotting and 'magic' commands. "
+                           "To learn more, check out the "
+                           f"<a href=\"{qtconsole_link}\">documentation</a>."
+                           f"<br><br>{button_text}"),
               'widgets': [sw.ipython_console],
-              'run': ["li = list(range(100))", "d = {'a': 1, 'b': 2}"]
+              'run': [
+                  "test_list_tour = list(range(100))",
+                  "test_dict_tour = {'a': 1, 'b': 2}",
+                  ]
               },
 
-             {'title': _("The Variable Explorer"),
+             {'title': _("Variable Explorer"),
               'content': _("In this pane you can view and edit the variables "
                            "generated during the execution of a program, or "
-                           "those entered directly in one of Spyder "
-                           "consoles.<br><br>"
-                           "As you can see, the Variable Explorer is showing "
-                           "the variables generated during the last step of "
-                           "this tour. By doing a double-click on any "
-                           "of them, a new window will be opened, where you "
+                           "those entered directly in the Console.<br><br>"
+                           "If you ran the code in the previous step, "
+                           "the Variable Explorer will show "
+                           "the list and dictionary objects it generated. "
+                           "By double-clicking any variable, "
+                           "a new window will be opened where you "
                            "can inspect and modify their contents."),
               'widgets': [sw.variable_explorer],
               'interact': True},
 
              {'title': _("Help"),
               'content': _("This pane displays documentation of the "
                            "functions, classes, methods or modules you are "
-                           "currently using in the Editor or the Consoles.<br><br>"
-                           "To use it, you need to press <b>Ctrl+I</b> in "
-                           "front of an object. If that object has some "
-                           "documentation associated with it, it will be "
-                           "displayed here."),
+                           "currently using in the Editor or the Console."
+                           "<br><br>To use it, press <b>Ctrl+I</b> "
+                           "(<b>Cmd-I</b> on Mac) with the text cursor "

```

Comment:
```suggestion
                           "(<b>Cmd-I</b> on macOS) with the text cursor "
```

That's the right name for that OS.

Reply:
macOS is the name of the OS, but Mac (short for Macintosh) is the name of the computer platform. I.e. I own a Mac that runs macOS 10.15. I would have used macOS here, but I wanted to be consistent with the other place I used it above, which would have required an extra line break to change from Mac which I judged wasn't worth the trouble. However, since that latter usage will be eliminated per your comment, there's no harm in applying this here.